src/Illuminate/Foundation/Auth/ThrottlesLogins.php

```diff
@@ -112,7 +112,7 @@ public function maxAttempts()
     /**
      * Get the number of minutes to throttle for.
      *
-     * @return int
+     * @return int|array
```

Comment:
It kind of seems weird that the function can return an `int` or an `array`. I understand why you've had to change this, but it feels messy. It means you would always need to do a check on the return.

I'm not sure what the solution is; maybe always wrap the response in an array? But that would make it a breaking change (although probably a small impact).